suite/tests/client-interface/drreg-test.dll.c

```diff
@@ -265,6 +265,16 @@ event_app_instruction(void *drcontext, void *tag, instrlist_t *bb, instr_t *inst
             res = drreg_unreserve_aflags(drcontext, bb, inst);
             CHECK(res == DRREG_SUCCESS, "unreserve of aflags should work");
         }
+    } else if (subtest == DRREG_TEST_6_C) {
+        /* Save the 3rd DR slot at the label, restore register after
+         * the xl8 point this test.
```

Comment:
I don't understand "the xl8 point this test": some words are missing?

Reply:
"in"